redisinsight/api/config/test.ts

```diff
@@ -3,4 +3,7 @@ export default {
     env: 'test',
     requestTimeout: 1000,
   },
+  profiler: {
+    logFileIdleThreshold: parseInt(process.env.PROFILER_LOG_FILE_IDLE_THRESHOLD, 10) || 1000 * 3, // 3sec
```

Comment:
Probably better move it to constant, like DEFAULT_LOG_FILE_THRESHOLD

Reply:
To move what? We are already in config file.